Source/ToolCore/JSBind/JSBTypeScript.cpp

```diff
@@ -57,6 +65,9 @@ String JSBTypeScript::GetScriptType(JSBFunctionType* ftype)
         scriptType = "string[]";
     }
 
+    if (ftype->type_->asHeapPtrType())
+        scriptType = "any";
+
```

Comment:
Ok, when I get a chance I'll run with that fix removed, it'll pop the example up immediately :)